genCode.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from io import open
+
+import json
+import os
+import re
+import six
+import textwrap
+
+from jinja2 import Environment, FileSystemLoader
+
+GENERATED_STRING = "# This file is generated!  Do not edit!"
+# ROUTING_KEY_WHITELIST is also defined in test/base/__init__.py
+# if you change it, please update it there as well.
+ROUTING_KEY_WHITELIST = ("name", "multipleWords", "constant")
+
+
+def loadJson(filename):
+    with open(filename) as fh:
+        return json.load(fh)
+
+
+def createInitPy(pydir, modules):
+    initpy = os.path.join(pydir, '__init__.py')
+    print(initpy)
+    with open(initpy, 'w') as fh:
+        print("#!/usr/bin/env python", file=fh)
+        print(GENERATED_STRING, file=fh)
+        for module in modules:
+            print("from taskcluster.sync.{module} import {module}".format(module=module),
+                  file=fh)
+        print("__all__ = [", file=fh)
```

Comment:
You're right, it's not really needed.
However, at the very least, it's suppressing lint warnings.  Given a choice between

* `# NOQA` or other lint skipping methods, or
* keeping `__all__`,

which do you prefer?  I prefer the latter.